desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.test.tsx

```diff
@@ -0,0 +1,175 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import PathBrowser from './PathBrowser';
+import { debug } from 'webpack';
+
+const handleFilePathChangeMock = jest.fn();
+
+const breadcrumbsTestConfig1 = [
+  {
+    url: 'abfs://',
+    label: 'abfs://'
+  },
+  {
+    url: 'abfs://test',
+    label: 'test'
+  },
+  {
+    url: 'abfs://test/test1',
+    label: 'test1'
+  }
+];
+
+const breadcrumbsTestConfig2 = [
+  {
+    url: 'abfs://',
+    label: 'abfs://'
+  },
+  {
+    url: 'abfs://test',
+    label: 'test'
+  },
+  {
+    url: 'abfs://test/test1',
+    label: 'test1'
+  },
+  {
+    url: 'abfs://test/test1/test2',
+    label: 'test2'
+  },
+  {
+    url: 'abfs://test/test1/test2/a very very very long test label',
+    label: 'a very very very long test label'
+  },
+  {
+    url: 'abfs://test/test1/test2/a very very very long test label/a very very very long test label 1',
+    label: 'a very very very long test label 1'
+  },
+  {
+    url: 'abfs://test/test1/test2/a very very very long test label/a very very very long test label 1/a very very very long test label 2',
+    label: 'a very very very long test label 2'
+  }
+];
+
+describe('Pathbrowser', () => {
+  describe('Pathbrowser breadcrumbs', () => {
+    test('renders the specified seperator to seperate the breadcrumbs', () => {
+      render(
+        <PathBrowser
+          handleFilePathChange={handleFilePathChangeMock}
+          breadcrumbs={breadcrumbsTestConfig1}
+          seperator={'%'}
+          showIcon
+        />
+      );
+      const seperator = screen.getAllByTestId('hue-path-browser-breadcrumb-seperator');
```

Comment:
As a rule of thumb, only use testId if you can't find the elements using the preferred way. If you can't use the preferred queries on component you have written yourself then consider improving the components. It is all connected, well written components are more semantic, have better accessibility and can easily be tested the way a user would interact with them :-)
 
Checkout https://testing-library.com/docs/queries/about#priority